1h+++ (#1074)
thewca/wca-regulations: wca-guidelines.md

- 1h++) [RECOMMENDATION] All attempts in the final round of an event, as well as all 3x3x3 Fewest Moves attempts, should have the same scramble sequences for all competitors (i.e. only 1 group).
- 1h1+) [CLARIFICATION] Scramblers/judges should only scramble for/judge other competitors in the same group if it is important for competition logistics.
- 1h1++) [CLARIFICATION] For 3x3x3 Fewest Moves, judges may judge competitors in their own group before they have finished their attempt (i.e. while they are competing).

> - 1h1+++) [ADDITION] Competitors are permitted to compete earlier/later than the scheduled time, at the discretion of the WCA Delegate.

Author: ```suggestion
- 1h1+++) [ADDITION] Competitors may request to compete earlier or later than the scheduled time for the round in a separate group. Competitors requesting such accommodations should contact the organization team and WCA Delegate before the competition.
```

> - 1h1+++) [ADDITION] Competitors may request to compete earlier or later than the scheduled time for the round in a separate group. Competitors requesting such accommodations should contact the organization team and WCA Delegate before the competition.

Author: ```suggestion
- 1h1+++) [ADDITION] Competitors may request to compete at a time different to the scheduled time for the round in a separate group, at the discretion of the WCA Delegate. Competitors requesting such accommodations should contact the organization team and WCA Delegate before the competition.
```

Reviewer: I'm not a fan of "earlier or later". I think something like "at a different time" would be better. Perhaps:
`Competitors may request to compete at a time different to the scheduled time...`

> - 1h1+++) [ADDITION] Competitors may request to compete at a time different to the scheduled time for the round in a separate group, at the discretion of the WCA Delegate. Competitors requesting such accommodations should contact the organization team and WCA Delegate before the competition.

Author: ```suggestion
- 1h1+++) [ADDITION] Competitors may request to compete at a time different to the scheduled time for the round in a separate group. The WCA Delegate should carefully consider the fairness of the situation before approving any such requests. Competitors requesting such accommodations should contact the organization team and WCA Delegate before the competition.
```

> - 1h1+++) [ADDITION] Competitors may request to compete at a time different to the scheduled time for the round in a separate group. The WCA Delegate should carefully consider the fairness of the situation before approving any such requests. Competitors requesting such accommodations should contact the organization team and WCA Delegate before the competition.

Reviewer 2: I'd add some explicit caveats that this can't break 9l of E2b++.

In particular, I think we should definitely exclude all FMC attempts from this. Theoretically, someone could start slightly early or slightly late without violating E2b++, but 1) "the slightly early" case would violate E2b++ if they decide to finish their attempt before everyone else has started, and 2) the "slightly late" case is already covered by E2b++.

```suggestion
- 1h1+++) [ADDITION] Competitors may request to compete at a time different to the scheduled time for the round in a separate group. The WCA Delegate should carefully consider the fairness of the situation before approving any such requests, and must ensure that such attempts follow [Regulation 9l](regulations:regulation:9l) (i.e. they must start after all competitors have finished their attempts from any previous round of the same event, and finish before any competitor has started an attempt for any following round of the same event). Competitors requesting such accommodations should contact the organization team and WCA Delegate before the competition. Exception: such accommodations are not permitted for 3x3x3 Fewest Moves, which specifies its own limited accommodations in [Guideline E2b++](guidelines:guideline:E2b++).
```

- 1h++) [RECOMMENDATION] All attempts in the final round of an event, as well as all 3x3x3 Fewest Moves attempts, should have the same scramble sequences for all competitors (i.e. only 1 group).
- 1h1+) [CLARIFICATION] Scramblers/judges should only scramble for/judge other competitors in the same group if it is important for competition logistics.
- 1h1++) [CLARIFICATION] For 3x3x3 Fewest Moves, judges may judge competitors in their own group before they have finished their attempt (i.e. while they are competing).

> - 1h1+++) [ADDITION] Competitors may request to compete at a time different to the scheduled time for the round in a separate group. The WCA Delegate should carefully consider the fairness of the situation before approving any such requests, and must ensure that such attempts follow [Regulation 9l](regulations:regulation:9l) (i.e. they must start after all competitors have finished their attempts from any previous round of the same event, and finish before any competitor has started an attempt for any following round of the same event). Competitors requesting such accommodations should contact the organization team and WCA Delegate before the competition. Exception: such accommodations are not permitted for 3x3x3 Fewest Moves, which specifies its own limited accommodations in [Guideline E2b++](guidelines:guideline:E2b++).

Reviewer 3: Any reason there's `must` in parentheses?

Edit: I mean, it should definitely be `must` for finishing, but not for starting maybe?

Author: I think in this case it's just a clarifying example and still works here
